### DISCUSSION TITLE
Goodtables Cloud Storage support

### DISCUSSION THREAD

**Discussion by vaa-nick-swann** [frictionlessdata/frictionlessdata.io#616] [Ideas]
Would be really useful if the goodtables services supported more cloud storage options, Gcp buckets and azure for instance

**Comment by cpina** [comment]
Just to add: I really like using `rclone` for interacting with some cloud storage options (usually for me object storage, but it supports a long list of backends, see https://rclone.org/overview/).

Some weeks ago I was reading how `restic` (a backup solution) integrated with `rclone`: https://restic.net/blog/2018-04-01/rclone-backend I don't know if this is out of scope for Goodtables but if desired that might be a possible way.
